docs/modules/Environment/met_installation.rst

```diff
@@ -7,128 +7,233 @@ Compiling MET
-
-If you wish to have the Python embedding functionality, you'll want to set the next three environment variables - MET_PYTHON, MET_PYTHON_CC, and MET_PYTHON_LD. MET_PYTHON should be set to the location containing the bin, include, lib and share subdirectories for Python.  In this case, we're using an anaconda location, but often it is in a location in /usr/local/python3.  MET_PYTHON_CC should be set to dash upper case I, "-I", followed by the directory containing the Python include files.  In this case, the value is ${MET_PYTHON}/include/python3.7m. You may be able to get this information by running "python3-config --cflags", however, in some cases, like on this machine the running the python3-config command provides additional information that is not necessary to include.
+The configuration file is named install_met_env.hera.  "Hera" is the name of
+the machine we will be working on.  Looking in the file, we see the first
+three lines contain "module use" and "module load" statements.  These are
+loading the intel compiler and the Python package via anaconda on this machine.
+If you are installing on a machine that does not use modules, be sure that the
+compiler executables, for example gcc, icc, etc., are in your PATH environment
+variable, and for Python embedding, make sure that the Python version 3.6.3 or
+higher executable is in your PATH environment variable as well.  The MET tools
+include the ability to embed Python to a limited degree.  Users may use Python
+scripts and whatever associated Python packages they wish in order to prepare
+2D gridded data fields, point observations, and matched pairs as input to the
+MET tools.  Our installation will include the Python embedding functionality.
+
+Next in the configuration file is TEST_BASE, which should be set to the
+installation directory - the full path, including the 10.0.1 directory you
+recently created.  
+
+Then, we will set the COMPILER.  The format here is the name of the compiler,
+followed by an underscore, followed by the version number.  In this case, we
+are using intel_18.0.5.274  because we're using version 18.0.5.274 of the intel
+compiler.  For the GNU family of compilers, use "gnu" for the compiler name.
+For the Intel family of compilers, use "intel", "ics", "ips" or another name,
+depending on your system.  For the PGI family of compilers, use "pgi" for the
+compiler name.  
+
+The next variable to set is the MET_SUBDIR, which is the location where the top
+level MET subdir, for example met-10.0.1, should be placed. I typcially set this
```

Comment:
typo with "I typcially set", should be "I typically set".

Reply:
Fixed.  Thank you so much for catching this error!